### ISSUE TITLE
[Form Recognizer] test_dac_analyze_custom_model and test_dmac_training failed in Public cloud

### ISSUE THREAD

**Issue by zedy-wj** [Azure/azure-sdk-for-python#23800]
We are running live Tests against other clouds like US Gov and Azure China Cloud. The goal is to check whether new azure sdk package work with other clouds or not.

**Error Description:**
[`test_dac_analyze_custom_model`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_custom_model.py#L212-L231) and [`test_dmac_training`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dmac_training.py#L243-L247) failed in Public cloud.

The latest result in weekly pipeline at here: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1464548&view=results.
Tthe same error appeared in nightly runs: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1476548&view=results.

**Error Track:**

1. [test_dac_analyze_custom_model](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_custom_model.py#L212-L231)
- Error message: 
```
2022-03-30T14:27:35.2160519Z >       assert result.documents[0].fields.get("FullSignature").value == "signed"
2022-03-30T14:27:35.2163807Z E       AssertionError: assert None == 'signed'
2022-03-30T14:27:35.2166383Z E         +None
2022-03-30T14:27:35.2168822Z E         -'signed'
```
2. [test_dmac_training](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dmac_training.py#L243-L247)

- Error message:
```
2022-03-30T14:27:35.2904206Z     @FormRecognizerPreparer()
2022-03-30T14:27:35.2907407Z     @DocumentModelAdministrationClientPreparer()
2022-03-30T14:27:35.2910672Z     @recorded_by_proxy
2022-03-30T14:27:35.2914241Z     def test_build_model_azure_blob_path_filter(self, client, formrecognizer_storage_container_sas_url, **kwargs):
2022-03-30T14:27:35.2917870Z         set_bodiless_matcher()
2022-03-30T14:27:35.2921037Z         with pytest.raises(HttpResponseError) as e:
2022-03-30T14:27:35.2924773Z             poller = client.begin_build_model(formrecognizer_storage_container_sas_url, "template", prefix="subfolder")
2022-03-30T14:27:35.2928397Z >           model = poller.result()
2022-03-30T14:27:35.2932335Z E           Failed: DID NOT RAISE <class 'azure.core.exceptions.HttpResponseError'>
```

**Expected behavior:**
Tests `test_dac_analyze_custom_model` and `test_dmac_training` can be successfully passed on `Prod`, `Canary` and `Public` clouds.

@benbp, @jameszliao-msft, @lmazuel, @meeraharidasa , @mayurid, @catalinaperalta for notification.

**Comment by catalinaperalta** [comment]
Hi @zedy-wj, I added a fix for these test issues last Friday (in #23781, #23790). If you run the pipeline again it should be clear of these issues now, please let me know if you're still seeing them in a new run.

**Comment by zedy-wj** [comment]
@catalinaperalta  - Thanks for the quick fix, we will close this issue.
